docs/conf.py

```diff
@@ -54,18 +92,44 @@
 # documents. Set to the "smart" one.
 default_role = 'obj'
 
-# -- Options for intersphinx extension ---------------------------------------
+# Disable having a separate return type row
+napoleon_use_rtype = False
+
+# Disable google style docstrings
+napoleon_google_docstring = False
+
+# -- Options for intersphinx extension -----------------------------------------
 
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {'https://docs.python.org/': None,
                        'http://aiohttp.readthedocs.io/en/stable': None,
                        'https://aioftp.readthedocs.io/': None}
 
-# -- Options for HTML output -------------------------------------------------
+# -- Options for HTML output ---------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+
+from sunpy_sphinx_theme.conf import *  # NOQA
 
-try:
-    from sunpy_sphinx_theme.conf import *  # NOQA
-except ImportError:
-    html_theme = 'alabaster'
+# Add any paths that contain custom static files (such as style sheets) here,
+# relative to this directory. They are copied after the builtin static files,
+# so a file named "default.css" will overwrite the builtin "default.css".
+# html_static_path = ['_static']
+
+# Render inheritance diagrams in SVG
+graphviz_output_format = "svg"
+
+graphviz_dot_args = [
+    '-Nfontsize=10',
+    '-Nfontname=Helvetica Neue, Helvetica, Arial, sans-serif',
+    '-Efontsize=10',
+    '-Efontname=Helvetica Neue, Helvetica, Arial, sans-serif',
+    '-Gfontsize=10',
+    '-Gfontname=Helvetica Neue, Helvetica, Arial, sans-serif'
+]
 
-html_theme_options = {'logo_url': 'https://parfive.readthedocs.io/en/latest/'}
+html_theme_options = {
+    'logo_url': 'https://parfive.readthedocs.io/en/latest/',
+    "page_toctree_depths": {}
```

Comment:
We default the theme to include this which is for the gallery. No gallery since the change in the theme has broken the doc build.